src/components/contact-manager/Create.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div class="contact-form">
+    <form @submit.prevent="validate">
+      <div class="row">
+        <div class="col-md-6">
+          <div class="row">
+            <div class="col-md-6 form-group">
+              <label for="firstname">First name</label>
+              <input
+                type="text"
+                name="firstname"
+                id="firstname"
+                placeholder="John"
+                class="form-control"
+                @change="elemChanged"
+                v-model="formValues.firstname.value"
+              />
+              <div class="error" v-if="errors.firstname">
+                {{ errors.firstname }}
+              </div>
+            </div>
+            <div class="col-md-6 form-group">
+              <label for="lastname">Last name</label>
+              <input
+                type="text"
+                name="lastname"
+                id="lastname"
+                placeholder="Doe"
+                class="form-control"
+                @change="elemChanged"
+                v-model="formValues.lastname.value"
+              />
+              <div class="error" v-if="errors.lastname">
+                {{ errors.lastname }}
+              </div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-md-12 form-group">
+              <label for="email">Email</label>
+              <input
+                type="text"
+                name="email"
+                id="email"
+                placeholder="johndoe@example.com"
+                class="form-control"
+                @change="elemChanged"
+                v-model="formValues.email.value"
+              />
+              <div class="error" v-if="errors.email">{{ errors.email }}</div>
+            </div>
+          </div>
+          <button type="submit" class="btn btn-success form-control">Save</button>
+        </div>
+      </div>
+    </form>
+  </div>
+</template>
+
+<script type="text/javascript">
+import omit from "lodash/omit";
+import { validatorNew } from "@/helpers/validate";
+
+export default {
+  name: "ContactCreate",
+  props: {
+    create: Function
```

Comment:
we have to emit a method instead of using callback function

Reply:
done.